src/utilities/gql.ts

```diff
@@ -2155,6 +2432,19 @@ const gql = {
     }
   `,
 
+  SUB_DERIVATION_GROUPS: `#graphql
+    subscription SubDerivationGroups {
+      models: ${Queries.DERIVATION_GROUP_COMP}(order_by: {name: desc}) {
```

Comment:
is `models` the correct name for this? It might be good to disambiguate from an actual mission model if a derivation group is also called a model.

Reply:
Definitely looks like an old copy/paste typo that never got cleaned up - changed to `derivationGroups`!